src/stackdriver_agent.c

```diff
@@ -146,39 +156,52 @@ static int sagt_read(user_data_t *user_data)
   // Stats for API requests. The corresponding uc_meta_data_set calls are in
   // write_gcm.c.
   {
-    uint64_t value;
-    if (uc_meta_data_get_unsigned_int(
-            &vl, SAGT_API_REQUESTS_SUCCESS, &value) == 0) {
-      sagt_submit_derive("api_request_count", "success",
-                         now, interval, value, NULL);
-    }
-    if (uc_meta_data_get_unsigned_int(
-            &vl, SAGT_API_REQUESTS_CONNECTIVITY_FAILURES, &value) == 0) {
-      sagt_submit_derive("api_request_count", "connectivity_failures",
-                         now, interval, value, NULL);
-    }
-    if (uc_meta_data_get_unsigned_int(
-            &vl, SAGT_API_REQUESTS_ERRORS, &value) == 0) {
-      sagt_submit_derive("api_request_count", "errors",
-                         now, interval, value, NULL);
+    meta_data_t *md = meta_data_create();
+    if (meta_data_add_string(
+            md, "stackdriver_metric_type",
+            AGENT_PREFIX "/api_request_count") == 0) {
+      uint64_t value;
```

Comment:
For consistency with the next block, you can keep this declaration outside of the conditional, or move the declarations of `throttling` and `streamspace_size` into the next conditional as well.

Reply:
I don't think this is inconsistent, or at least I'm having trouble understanding what would be consistent, given that the blocks do different things:
- This block is setting one metric with three time series;
- the block below is setting two metrics with one time series each.